configs/config.protectli_cml_vp4670

```diff
@@ -25,7 +26,6 @@ CONFIG_TPM_MEASURED_BOOT=y
 CONFIG_BOOTMEDIA_LOCK_CONTROLLER=y
 CONFIG_BOOTMEDIA_LOCK_WPRO_VBOOT_RO=y
 CONFIG_BOOTMEDIA_LOCK_IN_VERSTAGE=y
-CONFIG_BOOTMEDIA_SMM_BWP=y
```

Comment:
We keep it enabled for VP4630 and VP4650, but not VP4670?

Reply:
Of course not... Disabled SMM BWP for all CML platforms.